README.md

```diff
@@ -6,6 +6,12 @@ A Go HTTP client for the [MongoDB Atlas API](https://docs.atlas.mongodb.com/api/
 
 Note that `go-client-mongodb-atlas` only supports the two most recent major versions of Go.
 
+## Maintenance Update
+
+`go-client-mongodb-atlas` is currently under limited maintenance. 
+For new projects please use an official [atlas-sdk-go](https://github.com/mongodb/atlas-sdk-go)
+Please follow an official [migration guide](https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_3_migration.md) 
```

Comment:
A couple suggestions for wording.. mostly to make sure they know "currently under limited maintenance" doesn't mean it's temporary
```suggestion
`go-client-mongodb-atlas` is now deprecated and receives limited maintenance. 
For new projects, please use the official [Atlas Go SDK](https://github.com/mongodb/atlas-sdk-go).
To learn how to migrate from go-client-mongodb-atlas to the Atlas Go SDK, see the [migration guide](https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_3_migration.md).
```